driver-core/src/main/java/com/datastax/driver/core/schemabuilder/TableOptions.java

```diff
@@ -358,6 +360,19 @@ public T speculativeRetry(SpeculativeRetryValue speculativeRetry) {
     }
 
     /**
+     * Define whether or not change data capture is enabled on this table.
+     * <p/>
+     * If no call is made to this method, the default value set by Cassandra is {@code false}.
```

Comment:
Should we warn users that if they call this method and later on execute this statement against C* < 3.8 they will get an error?